wallet-new/src/Cardano/Wallet/API/V1/Types.hs

```diff
@@ -348,11 +342,11 @@ instance Arbitrary AccountUpdate where
   arbitrary = AccountUpdate . fromString <$> pure "myAccount"
 
 -- | A type incapsulating a password update request.
-data PasswordUpdate = PasswordUpdate
-  { -- | The old password.
+data PasswordUpdate = PasswordUpdate {
     pwdOld :: !Text
-    -- | The new password.
+    -- ^ The old password.
   , pwdNew :: !Text
+    -- ^ The new password.
```

Comment:
Don't know how I missed that in previous PRs: why `pwdOld` and `pwdNew` are not `SpendingPassword`s here?

Reply:
Excellent point, will amend.